docker-images/notebook/worker-template.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       - 6GB
       - --death-timeout
       - '60'
-    image: pangeo/worker:2018-05-17
+    image: pangeo/worker:2018-05-19
```

Comment:
Following @yuvipanda's advice after the latest fiasco, we decided to stop using dates for the tag and start using git commit hashes.